### PR TITLE
Teacher homepage: update strings

### PR DESCRIPTION
**Pull request by breville**
screenshot 2017-06-20 10 29 39

### CODE REVIEW

**Comment by breville** [comment]
cc @tanyaparker

**Comment by tanyaparker** [comment]
Right now we have it split "Recent Courses" on the teacher homepage and "My Courses" on the Courses page. We want all of the purple cards on the homepage with title "My Courses." Then remove all the purple course cards from Courses. I would like this string change to go out together with that so they don't think that their courses are only 2 if they happen to have more.

File: apps/src/templates/studioHomepages/TeacherHomepage.jsx [hunk 1]
```diff
@@ -45,7 +45,7 @@ const TeacherHomepage = React.createClass({
         <RecentCourses
           courses={courses}
           showAllCoursesLink={true}
-          heading={i18n.recentCourses()}
```
Comment (Erin007) [inline apps/src/templates/studioHomepages/TeacherHomepage.jsx]: This probably means recentCourses isn't used anywhere and we can delete that string?

**Comment by tanyaparker** [comment]
LGTM - thanks!

**Pull request merged**